pkg/registry/apis/query/header_utils.go

```diff
@@ -10,17 +10,20 @@ import (
 // Set of headers that we want to forward to the datasource api servers. Those are used i.e. for
 // cache control or identifying the source of the request.
 //
-// The headers related to grafana alerting can be found here:
-// https://github.com/grafana/grafana-ruler/blob/96e6d4b25c0d973a7615b92b35739511a6fbd72f/pkg/ruler/rulesmanager/ds_query_rule_evaluator.go#L313-L328
+// The headers related to grafana alerting (x-rule-*), should match the list at
+// https://github.com/grafana/grafana/blob/f8ae71e4583499dd461ebaed31451966be04220b/pkg/services/pluginsintegration/clientmiddleware/usealertingheaders_middleware.go#L23
```

Comment:
ideally we should have a shared `x-rule-*` list somewhere that we can just include.